bin/magi.rs

```diff
@@ -50,6 +51,9 @@ pub struct Cli {
     l2_engine_url: Option<String>,
     #[clap(long)]
     jwt_secret: Option<String>,
+    /// Path to a JWT secret to use for authenticated RPC endpoints
+    #[clap(long, global = true, required = false)]
```

Comment:
Does this need to be marked as global? I think that means it is valid for all subcommands, but there aren't any subcommands here so it doesn't really do anything?

Also I think optionals are by default not required, so we can remove the required flag.

Reply:
Agreed yes, I was mostly just following reth's precedent. We can probably refactor/improve these cli opts in a separate pr